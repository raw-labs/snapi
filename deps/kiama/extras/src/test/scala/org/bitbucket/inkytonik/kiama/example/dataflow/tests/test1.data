{
    y = v
    z = y
    x = v
    while (x)
    {
        x = w
        x = v
    }
    return x
}
