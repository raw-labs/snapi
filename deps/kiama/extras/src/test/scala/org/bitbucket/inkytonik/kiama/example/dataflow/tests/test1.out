Block(List(Assign(y,v), Assign(x,v), While(x,Block(List(Assign(x,v)))), Return(x)))
